common/templates/common/selection_drugs.html

```diff
@@ -1322,4 +1363,68 @@ <h4 class="mb-0">{{ title }}</h4>
     }
 }
 </script>
+<script>
+function compoundPreview(selector="a.struct") {
```

Comment:
Please move this function definition to a new or existing js file in the static folder and import it. I see this function is also used on another page. Best not to have copies of the function